### DISCUSSION TITLE
TERM environment variable not set.

### DISCUSSION THREAD

**Discussion by nuttaruj** [aahnik/tgcf#313] [General]
Hi, i deploy on heroku. 
log have message TERM environment variable not set.
how to fix. TERM > what value?

**Comment by tissole** [comment]
Probably you forgot a variable. Go to settings, config vars, and put all the values necessary following this [guide](https://github.com/aahnik/tgcf/wiki/Environment-Variables)

**Reply by azlk** [reply]
But there is not a single word about TERM var in this guide...:(
